heartbeat/CTDB.in

```diff
@@ -563,17 +565,6 @@ ctdb_start() {
 	rv=$?
 	[ $rv -ne 0 ] && return $rv
 
-	# Die if databases are corrupted
```

Comment:
Are we sure this wont make any issues for older versions of CTDB? We need to maintain compatibility with older versions too.

Reply:
Upstream Samba currently only supports 4.6+ (see https://wiki.samba.org/index.php/Samba_Release_Planning), but I checked back to 4.2 for the 00.ctdb event script. In 4.2, the 00.ctdb event script exists and includes logic to call "tdbtool check" (with tdbdump fallback) when initialised.